app/views/entitlements/_special_item_information.html.erb

```diff
@@ -142,7 +148,6 @@
         <h3>Allowed.</h3>
 
         <ul>
-          <li>You can ship alcohol as part of you total household good shipment, assuming that the alcohol you are shipping doesn’t violate any state-specific import laws.  You can check specific <%= link_to 'state and local laws', '#TODO' %> related to alcohol transportation for more information.</li>
           <li>You can ship alcohol as part of you total household good shipment, assuming that the alcohol you are shipping doesn’t violate any international customs or state-specific import laws. Check out <%= link_to 'federal US alcohol-related customs laws', '#TODO' %> and <%= link_to 'state-specific importation laws', '#TODO' %> for more details.</li>
```

Comment:
as part of you? part of _your_, I'm guessing.

Reply:
Also, there are still TODOs in this content. Can that be addressed?

Reply:
Also, it's probably household good _s_, not a single household good.

Reply:
@LillyMadigan @lfantano @jde1984 - do we have links for "federal US alcohol-related customs laws" and "state-specific importation laws"?